src/pages/Post/PostList/components/Posts.js

```diff
@@ -0,0 +1,47 @@
+import '../PostList.scss';
+import './Posts.scss';
+
+const Posts = ({ postData, formateDate }) => {
+  return (
+    <>
+      {postData.map((postData, index) => (
```

Comment:
다른 요소가 들어가지 않는 게 확실하다면, 이런 식으로 작성하시면 될 것 같습니다.
```suggestion
  return postData.map((postData, index) => (
  ...
```

Reply:
key prop에는 index의 사용을 지양해야 합니다. 대신 id와 같은 유니크한 값을 활용해 주세요!
참고 : [React 공식 문서](https://ko.legacy.reactjs.org/docs/lists-and-keys.html#keys)